parlai/agents/example_seq2seq/example_seq2seq.py

```diff
@@ -150,107 +145,82 @@ def share(self):
 
         return shared
 
-    def predict(self, xs, ys=None, is_training=False):
-        """Produce a prediction from our model.
-
-        Update the model using the targets if available.
-        """
+    def v2t(self, vector):
```

Comment:
We should document params, since this method has a complicated type signature. Or hide it with `_v2t`?